app/src/main/java/org/simple/clinic/home/report/ReportsScreen.kt

```diff
@@ -1,29 +1,67 @@
 package org.simple.clinic.home.report
 
+import android.annotation.SuppressLint
 import android.content.Context
 import android.util.AttributeSet
-import android.widget.LinearLayout
+import android.view.View
+import android.webkit.WebView
+import android.widget.FrameLayout
+import com.jakewharton.rxbinding2.view.RxView
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+import kotterknife.bindView
+import org.simple.clinic.R
+import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.util.visibleOrGone
+import org.simple.clinic.widgets.ScreenCreated
 import java.net.URI
+import javax.inject.Inject
 
-class ReportsScreen(context: Context, attrs: AttributeSet) : LinearLayout(context, attrs) {
+class ReportsScreen(context: Context, attrs: AttributeSet) : FrameLayout(context, attrs) {
 
   companion object {
     val KEY = ReportsScreenKey()
   }
 
+  @Inject
+  lateinit var controller: ReportsScreenController
+
+  private val webView by bindView<WebView>(R.id.reportsscreen_webview)
+  private val noReportView by bindView<View>(R.id.reportsscreen_no_report)
+
+  @SuppressLint("SetJavaScriptEnabled")
   override fun onFinishInflate() {
     super.onFinishInflate()
 
     if (isInEditMode) {
       return
     }
+
+    webView.settings.javaScriptEnabled = true
+
+    TheActivity.component.inject(this)
+
+    Observable.just(ScreenCreated())
+        .observeOn(Schedulers.io())
+        .compose(controller)
+        .observeOn(AndroidSchedulers.mainThread())
```

Comment:
Can you statically import `Schedulers.io()` and `AndroidSchedulers.mainThread()`? They look cleaner